lib/ui/widgets/patcherView/patch_selector_card.dart

```diff
@@ -58,7 +59,9 @@ class PatchSelectorCard extends StatelessWidget {
 
   String _getPatchesSelection() {
     String text = '';
-    for (final Patch p in locator<PatcherViewModel>().selectedPatches) {
+    for (final Patch p in locator<PatcherViewModel>()
+        .selectedPatches
```

Comment:
Why not sort when initializing `selectedPatches`?

Reply:
Why not sort when generating the json overall? that results in it being sorted everywhere, patch selector screen, patchcard, etc

Reply:
I tried that first, but it fails in this scenario.


https://github.com/ReVanced/revanced-manager/assets/11080523/c017d67d-7dec-467b-a663-3669e5e14e1b

BTW, I had written sort logic earlier in `loadLastSelectedPatches()` function of `PatcherViewModel` just above `notifyListeners()`. Is that the correct place?

Reply:
> Why not sort when generating the json overall? that results in it being sorted everywhere, patch selector screen, patchcard, etc

List is being sorted only for UI purpose and nothing else. That's why I think sorting it here in the view is correct. What do you think?

Reply:
In that case, I think your placement fits best